src/spot-client.ts

```diff
@@ -18,62 +17,62 @@ export class SpotClient extends BaseRestClient {
    * @param {AxiosRequestConfig} [requestOptions={}] HTTP networking options for axios
    */
   constructor(
-    key?: string | undefined,
-    secret?: string | undefined,
-    useLivenet: boolean = false,
-    restClientOptions: RestClientOptions = {},
-    requestOptions: AxiosRequestConfig = {}
+      key?: string | undefined,
+      secret?: string | undefined,
+      useLivenet: boolean = false,
+      restClientOptions: RestClientOptions = {},
+      requestOptions: AxiosRequestConfig = {}
   ) {
-    super(key, secret, getRestBaseUrl(useLivenet, restClientOptions), restClientOptions, requestOptions);
-
-    // this.requestWrapper = new RequestWrapper(
-    //   key,
-    //   secret,
-    //   getRestBaseUrl(useLivenet, restClientOptions),
-    //   restClientOptions,
-    //   requestOptions
-    // );
+    this.requestWrapper = new RequestWrapper(
+        key,
+        secret,
+        getRestBaseUrl(useLivenet, restClientOptions),
+        { ...restClientOptions, timePath: restClientOptions.timePath || "spot/v1/time", recv_window: 5000 },
+        requestOptions
+    );
     return this;
   }
 
-  async getServerTime(urlKeyOverride?: string): Promise<number> {
-    const result = await this.get('/spot/v1/time');
-    return result.serverTime;
+  getServerTime(): GenericAPIResponse {
+    return this.requestWrapper.get('spot/v1/time');
   }
 
   /**
    *
    * Market Data Endpoints
    *
-  **/
+   **/
 
-  getSymbols() {
-    return this.get('/spot/v1/symbols');
+
+  getSymbols(): Promise<APIResponse<SymbolInfo[]>> {
+    return this.requestWrapper.get('spot/v1/symbols');
```

Comment:
```suggestion
    return this.get('spot/v1/symbols');
```

Request wrapper should be deprecated, please revert this to `this.get('spot/v1....`